core/src/main/java/io/questdb/log/LogFactory.java

```diff
@@ -285,7 +284,8 @@ public void close(boolean flush) {
                                 // Keep running the job until it returns false to log all the buffered messages
                             }
                         } catch (Exception th) {
-                            // Exception means we cannot log anymore. Perhaps network is down or disk is full.
```

Comment:
Seems like your IDE formatter is a bit overzealus and wraps lines shorter than 120 chars (Intellij default).

Reply:
Ah yeah, I'm running VSCode.. I can fix this in my config and revert these changes